cmd/kyma/alpha/create/module/module.go

```diff
@@ -54,12 +53,15 @@ Build module modB in version 3.2.1 and push it to a local registry "unsigned" su
 		RunE:    func(_ *cobra.Command, args []string) error { return c.Run(args) },
 		Aliases: []string{"mod"},
 	}
-	cmd.Args = cobra.ExactArgs(3)
 
+	cmd.Flags().StringVar(&o.Version, "version", "", "Version of the module. This flag is mandatory.")
+	cmd.Flags().StringVarP(&o.Name, "name", "n", "", "Override the module name of the kubebuilder project. If the module is not a kubebuilder project this flag is mandatory.")
+	cmd.Flags().StringVarP(&o.Path, "path", "p", "", "Path to the module contents. (default current directory)")
+	cmd.Flags().StringVar(&o.ModPath, "mod-path", "./mod", "Specifies the target path where the module will be built into. If the path already has a descriptor use the overwrite flag to overwrite it. This flag shuld only be used by experienced users.")
```

Comment:
Perhaps renaming that flag to "target-path" or something more obvious is worth considering?

Reply:
I considered that name but it could easily be confused then with the previous flag (path).

It is nevertheless true that it is not very descriptive for someone that does not know the details of the command. Another option I considered is `mod-cache` because this folder kind of acts as a cache for the image building process.